public/app/plugins/panel/geomap/editor/LayerEditor.tsx

```diff
@@ -85,6 +87,36 @@ export const LayerEditor: FC<LayerEditorProps> = ({ options, onChange, data, fil
           showIf: (opts: MapLayerOptions) => opts.location?.mode === FrameGeometrySourceMode.Geohash,
           // eslint-disable-next-line react/display-name
           // info: (props) => <div>HELLO</div>,
+        })
+        .addFieldNamePicker({
+          path: 'location.lookup',
+          name: 'Lookup Field',
```

Comment:
All field options should use Sentence case (ie only capitalize first word)

Reply:
Addressed this for my PR. Most of the field options in geomap capitalize the first letter of every word—do you want met to address this in this PR or in a separate one?